README.md

```diff
@@ -20,14 +20,46 @@ To run in development (watch mode):
 npm run dev
 ```
 
-If you see warning `API key does not start with "SG.".`, make sure you've put a real SendGrid API key in `.env`.  Ask your teammates for more info.
+If you see warning `API key does not start with "SG.".`, make sure you've put a real SendGrid API key in `.env`. Ask your teammates for more info.
 
 To run in production:
 
 ```sh
 npm start
 ```
 
+## JS Docs/ Better-Docs
+
+---
+
+'JS Doc' is used to document this project.
+
+We would like to keep the project documentation up to date with changes to the main branch. For a pull request to be approved all functions should include a 'JS doc' block.
```

Comment:
```suggestion
JSDoc is used to document this project.

We would like to keep the project documentation up to date with changes to the main branch. For a pull request to be approved all functions should include a JSDoc block.
```

💭 For consistency with the rest of the documentation, and also matches how it's spelt on the JSDoc website.